demo/src/app/components/+tabs/demos/select-event/select-event.html

```diff
@@ -0,0 +1,7 @@
+<div class="mb-3">
+  <pre class="card card-block card-header" *ngIf="value">Event select is fired. Your choice is: {{value}}</pre>
+</div>
+<tabset>
+  <tab heading="04.05.2018" (select)="onSelect($event)">04.05.2018</tab>
```

Comment:
remove these dates and replace them with headings like `First tab` and `Second tab`

Reply:
also, change tab content to something more generic like here https://github.com/valor-software/ngx-bootstrap/pull/3652/files#diff-cf724fc338926969507b9d22f7ce773f